lib/elastic_apm/grape.rb

```diff
@@ -27,9 +25,18 @@ def start(app, config)
       config.logger.error format('Failed to start: %s', e.message)
       config.logger.debug "Backtrace:\n" + e.backtrace.join("\n")
     end
+    # rubocop:enable Metrics/MethodLength, Metrics/AbcSize
 
     private
 
+    def set_config(app, config)
+      config.service_name ||= app.name
+      config.framework_name ||= 'Grape'
+      config.framework_version ||= ::Grape::VERSION
+      config.logger ||= app.logger
```

Comment:
Can this be `nil`? Does Grape apps have a default logger?

Reply:
It says [here](https://github.com/ruby-grape/grape/blob/master/README.md#logging) that 

> Grape::API provides a logger method which by default will return an instance of the Logger class from Ruby's standard library.

So I think it's ok, no?

Reply:
Yes, very ok 😊